Zend/zend_compile.h

```diff
@@ -507,6 +507,7 @@ struct _zend_execute_data {
 #define ZEND_CALL_FAKE_CLOSURE       (1 << 23)
 #define ZEND_CALL_GENERATOR          (1 << 24)
 #define ZEND_CALL_DYNAMIC            (1 << 25)
+#define ZEND_CALL_SYNTAX_PIPE        (1u << 26u)
```

Comment:
knee-jerk reaction: "why not simply `(1 << 26)`, following the lines above?"
(below is `(1u << 31)` only because `(1 << 31)` would be a signed overflow [assuming 32-bit int])